libs/api-sdk/src/apiSdk.ts

```diff
@@ -133,6 +135,21 @@ export default class ApiSdk {
         await addMemberByApiKey(this._config, groupId, memberId, apiKey)
     }
 
+    /**
+     * Adds several members to a group using an API Key.
+     * @param groupId Group id.
+     * @param memberIds Member ids.
+     * @param apiKey API Key.
+     * @returns undefined.
```

Comment:
This is not 100% correct since a `void` function returns `undefined` but a `Promise<void>` resolves to an `undefined`. 

Therefore, you should write something like

```js
@returns {Promise<void>} a Promise that resolves to `undefined`.
```

Reply:
In my opinion, yes, it is better, but I was wondering if it would be possible to avoid the tag altogether in this case and the like. Just because, in essence, we don't return anything at all. The PR has been successfully merged but I just wanted to emphasize this fact. Thanks for bringing this conversation up Vivi!

Reply:
Thanks for the answer. I think it is worth showing what devs can expect from that function.